### PR TITLE
Create delete function for VA BRI

### PR DESCRIPTION
**Pull request by budiariyanto**
## What does this PR do?

[ARJ-90](https://kitabisa.atlassian.net/browse/ARJ-90)

## Why are we doing this? Any context or related work?
We need to create this function so client able to request VA deletion to
BRI

## Where should a reviewer start?
--

## Manual testing steps?
--

## Screenshots
No screenshot needed.

## Config changes
No config changed.

## Database changes
No database changed.

## Deployment instructions
Usual deployment.

### CODE REVIEW

File: core.go [hunk 1]
```diff
@@ -125,3 +125,26 @@ func (gateway *CoreGateway) GetReportVA(token string, req GetReportVaRequest) (r
 
 	return
 }
+
+func (gateway *CoreGateway) DeleteVA(token string, req DeleteVaRequestData) (res VaResponse, err error) {
+	token = "Bearer " + token
+	method := "DELETE"
+	body, err := json.Marshal(req)
```
Comment (abhisuta) [inline core.go:132]: Mas @budiariyanto, the body for `delete` endpoint shouldn't be in json format. The example is `institutionCode=J104408&brivaNo=77777&custCode=123456789115`.

File: request.go [hunk 2]
```diff
@@ -83,3 +83,9 @@ type DeleteCardTokenRequest struct {
 type DeleteCardTokenRequestData struct {
 	CardToken string `json:"card_token"`
 }
+
+// DeleteVaRequestData defines data payload for VA account deletion
+type DeleteVaRequestData struct {
+	InstitutionCode string `json:"institutionCode"`
+	CustCode        string `json:"custCode"`
+}
```
Comment (abhisuta) [inline request.go:91]: Sorry, but you are missing `BrivaNo` mas @budiariyanto

File: core.go (hunk 1 above)
Comment (budiariyanto) [reply core.go:132]: wah iya. Thanks mas

File: request.go (hunk 2 above)
Comment (budiariyanto) [reply request.go:85]: eh ini kok belum kedelete

Comment (budiariyanto) [reply request.go:85]: udah deng

**Pull request merged**